deploy/cluster_role.yaml

```diff
@@ -55,7 +55,7 @@ rules:
 - apiGroups:
   - gcp.managed.openshift.io
   resources:
-  - '*'
+  - projectclaims
```

Comment:
do you think that is sufficient? Were you able to test this?
The status of a CR is listed under a different resource than the CR spec.
See f.e.:   https://github.com/openshift/route-monitor-operator/blob/20cc01df4b3024039c3c90748b263d27155ceab7/config/rbac/role.yaml#L83

Reply:
$ oc auth can-i get projectclaims/status --as system:serviceaccount:gcp-project-operator:gcp-project-operator --as-group gcp.managed.openshift.io
yes